### DISCUSSION TITLE
Ability to specify attributes for approval

### DISCUSSION THREAD

**Discussion by gerb-ster** [cjmellor/approval#44] [Ideas]
I think it would be useful if one could specify which attributes of a model should be approved and which should be updated without approval. Sometimes you only want to approve a specific attribute.

**Comment by cjmellor** [comment]
It's a good idea. I'm open to PR's for it, but might look into it myself down the road also

**Reply by gerb-ster** [reply]
I've opened a PR for this:

https://github.com/cjmellor/approval/pull/46

**Reply by cjmellor** [reply]
Cool, I'll keep an eye on it.
